src/test/java/com/epam/validator/ValidatorTest.java

```diff
@@ -1,3 +1,65 @@
 package com.epam.validator;
 
-public class ValidatorTest { }
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.time.LocalDate;
+
+public class ValidatorTest {
+
+    @Test(expected = NullPointerException.class)
+    public void checkEmptyThrowException(){
+        String emptyString = "";
+        Validator.checkEmpty(emptyString);
+    }
+
+    @Test(expected = Exception.class)
+    public void checkDateDifferentThrowException(){
+        LocalDate firstDate = LocalDate.of(2020,12,5);
+        LocalDate secondDate = LocalDate.of(2018,12,5);
+        Validator.checkDateDifferent(firstDate,secondDate);
```

Comment:
How check method can throw exception???? It can return true/false values only

Reply:
Done